src/pe/options.rs

```diff
@@ -0,0 +1,12 @@
+#[derive(Debug, Copy, Clone)]
+pub struct ParseOptions {
+    pub rva: bool,
```

Comment:
the field for this seems a little misleading; isn't it more like `resolve_rva`, or even `not_memory_mapped` (latter not preferable because it encodes booleanness, but i think you get the point?)